command/acl_auth_method_create.go

```diff
@@ -50,8 +50,8 @@ ACL Auth Method Create Options:
     between 1-128 characters and is a required parameter.
 
   -type
-    Sets the type of the auth method. Currently the only supported type is
-    'OIDC'.
+    Sets the type of the auth method. Currently the only supported types are
+    'OIDC' and 'JWT'.
```

Comment:
```suggestion
    Sets the type of the auth method. Supported types are 'OIDC' and 'JWT'.
```

Same for `update`. We also need to update the [website](https://github.com/hashicorp/nomad/blob/main/website/content/docs/commands/acl/auth-method/create.mdx?plain=1#L32-L33).